tests/integration/ambient/istioctl_test.go

```diff
@@ -39,17 +38,20 @@ func TestZtunnelConfig(t *testing.T) {
 		Run(func(t framework.TestContext) {
 			// Test setup
 			istioCtl := istioctl.NewOrFail(t, istioctl.Config{})
-			istioCfg := istio.DefaultConfigOrFail(t, t)
 			g := NewWithT(t)
-			ztunnelPods, err := kubetest.NewPodFetch(t.AllClusters()[0], istioCfg.SystemNamespace, "app=ztunnel")()
+			ztunnelNS, err := locateDaemonsetNS(t, "ztunnel")
```

Comment:
Instead of discovering the ztunnel namespace, do we have any config param in the test framework (i.e., t.Settings().xxx) that we can use?

Reply:
Since platforms like [GKE and OpenShift](https://istio.io/latest/docs/ambient/install/platform-prerequisites/) install some components outside the default namespace, it might be worth exploring support for specifying the namespace in the test framework. Do you think that would require a lot of changes?

Reply:
It's not a straight forward path.

When I'm executing istio integration tests on Openshift, all istio components are being deployed into a single `istio-system` namespace. And that's completely fine. It's expected.

But when I'm using external control plane deployment for istio [1](https://github.com/istio/istio/blob/master/pkg/test/framework/components/istio/kube.go#L288-L295), it's hard to predict where istio components are being deployed.
That's why I'm trying to make it a bit more generic.

Reply:
Got it. Looping @kfaseela if she has any suggestions.